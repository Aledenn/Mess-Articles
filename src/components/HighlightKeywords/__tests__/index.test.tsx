import React from 'react';
import { render } from '@testing-library/react';
import HightlightKeywords from '..';

describe('HighlightKeywords', () => {
  test('can render', () => {
    const { container } = render(<HightlightKeywords words="正常使用" />);
    expect(container).toMatchSnapshot();
  });

  test('className', () => {
    const { container } = render(<HightlightKeywords className="cbsama" words="换个类名" />);
    expect(container).toMatchSnapshot();
  });

  test('keywords', () => {
    // const { cantainer } = render(
    //   <HightlightKeywords words="这是一个关键词" keywords={['关键词', '这是']} />,
    // );
  });
});
