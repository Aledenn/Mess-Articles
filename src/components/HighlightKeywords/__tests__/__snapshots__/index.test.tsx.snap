// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HighlightKeywords can render 1`] = `
<div>
  <div
    class="sc-bdVaJa kOFPiE tupu-input"
  >
    正常使用
  </div>
</div>
`;

exports[`HighlightKeywords className 1`] = `
<div>
  <div
    class="sc-bdVaJa kOFPiE tupu-input"
  >
    换个类名
  </div>
</div>
`;

exports[`HighlightKeywords keywords 1`] = `
<div>
  <div
    class="sc-bdVaJa kOFPiE"
  >
    <span />
    <span
      class="highlight"
    >
      这是
    </span>
    <span>
      一个
    </span>
    <span
      class="highlight"
    >
      关键词
    </span>
    <span />
  </div>
</div>
`;
